README.md

```diff
@@ -1 +1,208 @@
-# math-magician
\ No newline at end of file
+# Math Magician
+<a name="readme-top"></a>
+<div align="center">
+
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" /> -->
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [MathMagician](#MathMagician)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[MathMagician\] ](#-MathMagician-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+![](https://media.giphy.com/media/3F6V4WfEBeHsdTDcu3/giphy.gif)
+
```

Comment:
## Suggested Change

- [ ] Kindly remove this GIF file here as it's a different project. You're working on the Math-magician app, you can add the app screenshots here later.